corehq/pillows/xform.py

```diff
@@ -163,6 +164,22 @@ def get_couch_form_reindexer():
     )
 
 
+def get_form_kafka_pillow(pillow_id='form-processor-pillow'):
+    checkpoint = PillowCheckpoint(
+        pillow_id,
+    )
+    form_processor = FormProcessor()
+    return ConstructedPillow(
+        name=pillow_id,
+        checkpoint=checkpoint,
+        change_feed=KafkaChangeFeed(topics=[topics.FORM], group_id='domains-to-es'),  # TODO: include multiple topics
```

Comment:
Handling multiple topics is easy. See ledger pillow as an example.

Not sure if we're publishing couch forms to kafka yet though. But that isn't hard either.  See DomainDbKafkaPillow.